src/components/SocialsBar/WSocialsBar.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div
+    ref="socials"
+    class="opacity-60 hover:opacity-100 grid grid-rows-1 grid-flow-row content-center gap-y-md w-fit justify-items-center"
+  >
+    <IconExternalLink :url="socialsUrls.x">
+      <WIcon name="x" class="icon h-auto w-md" />
+    </IconExternalLink>
+    <IconExternalLink :url="socialsUrls.telegram">
+      <WIcon name="telegram" class="icon h-auto w-md" />
+    </IconExternalLink>
+    <IconExternalLink :url="socialsUrls.discord">
+      <WIcon name="discord" class="icon h-auto w-md" />
+    </IconExternalLink>
+    <IconExternalLink :url="socialsUrls.github">
+      <WIcon name="github" class="icon h-auto w-md" />
+    </IconExternalLink>
+
+    <!-- TODO(#413): set width to 40px to avoid wrong alignment -->
+    <div class="flex mt-3xl w-xl">
+      <p class="text-sm font-light text-white-400 [writing-mode:vertical-lr] -rotate-180 subtitle">
+        Enjoy Smart, Neutral,
+      </p>
+      <p class="text-sm font-light text-white-400 [writing-mode:vertical-lr] -rotate-180 subtitle">
+        and Secure Data
+      </p>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+// import { ref } from 'vue'
+
+import WIcon from '../icon/WIcon.vue'
+// import IconTwitter from '../icons/IconTwitter.vue'
+// import IconTelegram from '../icons/IconTelegram.vue'
+// import IconDiscord from '../icons/IconDiscord.vue'
+// import IconGithub from '../icons/IconGithub.vue'
+import IconExternalLink from './IconExternalLink.vue'
+import { socialsUrls } from './WSocialsBar'
+
+// const socials = ref(null)
```

Comment:
Delete commented code
```suggestion
// const socials = ref(null)
```

Reply:
Added TODO #20